compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -1953,3 +1953,6 @@ error.module.generated.for.client.decl.must.have.a.client.object.type=\
 
 error.module.generated.for.client.decl.cannot.have.mutable.state=\
   a module generated for a client declaration cannot have mutable state
+
+error.duplicate.default.values.for.key.in.table.literal=\
+  duplicate default values for key found in table row key(''{0}'')
```

Comment:
This doesn't quite reflect the issue, right? We need to say the default is used in more than one member which could result in a duplicate.

Reply:
Will depend on https://github.com/ballerina-platform/ballerina-lang/pull/37491#issuecomment-1290383442 anyway.